### PR TITLE
Update the iex output

### PR DESCRIPTION
**Pull request by romenigld**
I updated the iex output.
And was need to put the match for `person =  %Friends.Person{...}`
for after call the ´person.name` and others

### CODE REVIEW

File: pt/lessons/ecto/basics.md [hunk 1]
```diff
@@ -208,15 +208,26 @@ pessoa:
 
 ```elixir
 iex> %Friends.Person{}
-%Friends.Person{age: 0, name: nil}
+%Friends.Person{
```
Comment (SophieDeBenedetto) [inline pt/lessons/ecto/basics.md]: I'm not sure this output is correct. If we open up `iex` and simply create a new empty `Friends.Person` struct, then the resulting struct would not have any values for `name` or `age`

Comment (romenigld) [reply pt/lessons/ecto/basics.md]: The output of the struct is this, but the results are differents.
```
%Friends.Person{
  __meta__: #Ecto.Schema.Metadata<:built, "people">,
  age: 0,
  id: nil,
  name: nil
}
```

Comment (SophieDeBenedetto) [reply pt/lessons/ecto/basics.md:211]: Right, so shouldn't the output you added below have `id: 0` and name `""`?

Comment (romenigld) [reply pt/lessons/ecto/basics.md:211]: This was the result of my terminal.
with the elixir  1.10.0 version.

Comment (SophieDeBenedetto) [reply pt/lessons/ecto/basics.md:211]: Ah okay, do you mean to say your output is this?

```elixir 
%Friends.Person{
  __meta__: #Ecto.Schema.Metadata<:built, "people">,
  age: 0,
  id: nil,
  name: nil
}
```

If so, that makes sense and we should use that output on the lines below. But currently you have the name set as `"Tom"` on line 215 which I think is incorrect, it should be `nil`?

Comment (romenigld) [reply pt/lessons/ecto/basics.md:211]: yes it should be name: nil. Sorry!

Comment (SophieDeBenedetto) [reply pt/lessons/ecto/basics.md:211]: no prob! Look good!

**Pull request merged**